ui/desktop/app/routes/scopes.js

```diff
@@ -23,7 +23,6 @@ export default class ScopesRoute extends Route {
     try {
       const scopesCheck = await adapter.query(this.store, scopeSchema, {
         page_size: 1,
```

Comment:
Totally for learning purposes: I was checking the `adapter.query` [object documentation](https://api.emberjs.com/ember-data/5.3/classes/%3CInterface%3E%20Adapter/methods/query?anchor=query). I do understand `page_size` and `recursive` are params that belong to the`query`, is that correct?

Reply:
Correct. In this instance, I am simply removing recursive so it only fetches the scopes at the global level. Which is this case, will always be the `global` scope. I had added `recursive: true` prior because that is simply what we typically do when fetching all scopes but due to permission configurations, the user could make that fetch fail by not allowing all scopes to be visible to unauthenticated users (which is completely valid for them to do).